README.md

```diff
@@ -1,5 +1,25 @@
 # azure-hanaonazure-cli-extension
 
+# Install
+
+To install this extension just use the CLI extension add command
+
+```
+az extension add --source $LATES_RELEASE
+```
+
+# Usage
+
+To list all hanainstances for the subscription
+```
+az hanainstance list
+```
+
+To show details about a specific hana instance
```

Comment:
nit: add a colon `:` to the end of this line to demonstrate the command is below